sklearn/metrics/cluster/unsupervised.py

```diff
@@ -288,6 +289,14 @@ def calinski_harabaz_score(X, labels):
             (intra_disp * (n_labels - 1.)))
 
 
+@deprecated("Function 'calinski_harabaz_score' was renamed to "
+            "'calinski_harabasz_score' "
+            "in version 0.21 and will be removed in release 0.23. "
+            "The keyword normalize is no longer supported.")
+def calinski_harabaz_score(X, labels, normalize=False):
```

Comment:
What's happening here? We do not have ``normalize`` parameter now, right?

Reply:
The normalize parameter is not used in the computation of the score. It is a redundant parameter. It is present in the function definition of the deprecated function. It will no longer be a part of the revised function definition